### DISCUSSION TITLE
omitting fields with - operator

### DISCUSSION THREAD

**Discussion by lukepighetti** [pocketbase/pocketbase#6189] [Q&A]
It would be great if we could omit fields with the `-` operator. example:

record has fields: small, medium, large

you can specify `fields: "small,medium"`, but if you have many columns and need to omit only one due to size, it becomes cumbersome and error prone to keep up to date

if you could instead specify `fields: "-large"` it would make this much more manageable

note: the operator would only apply if there were no positive matches. ie: `"small"` and `"small,-large"` would be identical

**Comment by ganigeorgiev** [comment]
The current `fields` parameter acts as "include"-type of selection. Mixing it with another notation that does the opposite seems wrong to me.

For me it is confusing and unnecessary and for now it remains out of the scope. If there is users demand for it or a more prominent use case showup we can eventually introduce a new `exclude` or other similar parameter (_or even as modifier for the wildcard `*`, e.g. `*:exclude(field1,field2)`_).
